extensions/milight/libmilight.m

```diff
@@ -63,9 +63,9 @@ int milight_cacheCommands(lua_State *L) {
     pushCommand(L, "orange", 0xA0);
     pushCommand(L, "red", 0xB0);
     pushCommand(L, "pink", 0xC0);
-    pushCommand(L, "fuscia", 0xD0);
+    pushCommand(L, "fuchsia", 0xD0);
```

Comment:
People misspell this so badly that Google Sheets doesn't pick the correct suggestion...